build.gradle

```diff
@@ -124,6 +124,9 @@ ext {
     jets3tVersion = '0.7.1'
     awsJavaSdkVersion = '1.8.3'
 
+    // Our components
```

Comment:
In case we would to the separation as suggested bellow, I would move this configuration there as well. ( No other project is using this )